db/deploy/computed_columns/application_form_data.sql

```diff
@@ -0,0 +1,16 @@
+-- Deploy ccbc:computed_columns/application_form_data to pg
+
+begin;
+
+create or replace function ccbc_public.application_form_data(application ccbc_public.application) returns jsonb as
```

Comment:
```suggestion
create or replace function ccbc_public.application_form_data(application ccbc_public.application) returns ccbc_public.form_data as
```

having a computed column that returns the entire record instead of only the jsonb column would be preferable. You wouldn't need `application_form_data_id`, or other functions for the fields that get added later on